frontend/src/components/Header.jsx

```diff
@@ -70,6 +70,11 @@ const Header = () => {
                   tabIndex={0}
                   className='menu dropdown-content dropdown-main-brand-green'
                 >
+                  {userState.is_admin && (
```

Comment:
While this achieves the same, there's in user context `isAdmin`, which returns exactly that, but hiding away the non-JS name (`is_admin`) returned from api.